examples/fentry/bpf/fentry_example.c

```diff
@@ -0,0 +1,70 @@
+#include "common.h"
+#include "bpf_helpers.h"
+#include "bpf_tracing.h"
+#include "bpf_endian.h"
```

Comment:
We could avoid this dependency, see my comment below.

Reply:
bpf_endian.h removed.